org.dpsoftware.FireflyLuciferin.json

```diff
@@ -0,0 +1,100 @@
+{
+  "id": "org.dpsoftware.FireflyLuciferin",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "FireflyLuciferin",
+  "finish-args": [
+    "--share=network",
```

Comment:
why is network needed?

Reply:
please let me know if you need more details.